x-team/xp-ui
# Update props of JobApplicationCard

**Release Type:** *Non-breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3439

## Description

The JobApplicationCard now gets the `message` prop with the text copy already processed by XP-Reg. This component no longer deals with business logic for application date, and instead just displays the message.

## Checklist


## Related PRs

branch | PR
------ | ------
`XP-3439-job-application-date` | [XP Registration](https://github.com/x-team/xp-registration/pull/104)
`XP-3439-job-application-date` | [XP API](https://github.com/x-team/xp-api/pull/721)

## Steps to Test or Reproduce

- Go to:
    - https://deploy-preview-471--xpui-develop.netlify.com/?path=/story/ui-components-jobapplicationcard-states--withdraw
    - http://deploy-preview-471--xpui-develop.netlify.com/?path=/story/ui-components-jobapplicationcard-states--withdraw-is-saving
- Observe that the message appears as expected

## Impacted Areas in Application

- `src/components/ui/JobApplicationCard.js`

## Screenshots

<img width="366" alt="Screen Shot 2019-11-06 at 11 18 43" src="https://user-images.githubusercontent.com/131859/68305886-38b21000-0087-11ea-992c-218b6ef7883f.png">
